docs/getting-started-guides/docker-multinode.md

```diff
@@ -74,54 +78,165 @@ This pattern is necessary because the `flannel` daemon is responsible for settin
 all of the Docker containers created by Kubernetes.  To achieve this, it must run outside of the _main_ Docker daemon.  However,
 it is still useful to use containers for deployment and management, so we create a simpler _bootstrap_ daemon to achieve this.
 
-You can specify k8s version on very node before install:
+## Starting a Cluster
 
-```
+An example cluster is listed below:
+
+| IP Address  |   Role   |
+|-------------|----------|
+|10.10.102.152|   node   |
+|10.10.102.150|  master  |
+
+Optional, you can specify version before deployment, otherwise, we'll use latest `hyperkube` release as default.
```

Comment:
What is the latest `hyperkube` version? 
Is it automatically created per tag?
At least `v1.2.0-alpha.3` wasn't downloadable from `https://storage.googleapis.com/kubernetes-release/release/v1.2.0-alpha.3/bin/linux/amd64/hyperkube`